README.md

```diff
@@ -5,8 +5,9 @@ Automate and randomize the process of scheduling time with friends.
 
 The most basic action of this app is contacting people with varying frequencies to initiate a social event of some kind.
 	Example:
-		*User picks a frequency, the days and times that work, the lead time, appropriate time of day to contact, how many options to give (date and time combinations), and activities to choose from.
+	
+	*User picks a frequency, the days and times that work, the lead time, appropriate time of day to contact, how many options to give (date and time combinations), and activities to choose from.
 
-		*The app will contact friend (at an appropriate time) about hanging out (lead time) before the first (date and time configuration) to see if they are interested in doing (activity)
+	*The app will contact friend (at an appropriate time) about hanging out (lead time) before the first (date and time configuration) to see if they are interested in doing (activity)
 
-		*Friend does not need to have the app becuase they are getting contacted through text or email. If they want to download the app the can see the activity by joining and verifying all the contact info that people use to contact them. 
+	*Friend does not need to have the app becuase they are getting contacted through text or email. If they want to download the app the can see the activity by joining and verifying all the contact info that people use to contact them. 
```

Comment:
also, I think you'll have to remove the indentation.  Because it's indented, it thinks it's a codeblock.